README.md

```diff
@@ -230,6 +230,59 @@ Our indexation engine handles:
 
 > We include a Roslyn analyzer, detailed in a section below, that encourages developers to adorn classes that consume MongoDB collections with attributes (`IndexByAttribute` or `NoIndexNeededAttribute`). The aim is to increase awareness about which indexes should be used (or created) when querying MongoDB collections.
 
+### Support for inheritance
+
+The indexer mechanism support document inheritance and different indexer for a same collection. For example:
+
+```csharp 
+
+[BsonKnownTypes(typeof(DogPersonDocument), typeof(DogPersonDocument))]
```

Comment:
Isn't it weird that your base document has knowledge of it's child document?  Also feels weird that there's no full representation of the underlying collection, it's a mix of domain objects combined with each their db indexes.

Anyway I think this goes beyond the scope of this PR

Reply:
It's the recommended approach when having polymorphisme of document in Mongo: https://mongodb.github.io/mongo-csharp-driver/2.8/reference/bson/mapping/polymorphism/